src/lib/toolbox.rs

```diff
@@ -155,23 +155,6 @@ impl Toolbox {
                 Err(err) if err.downcast_ref::<NoResp>().is_some() => {
                     return;
                 }
-
-                Err(err0) if err0.downcast_ref::<tokio_postgres::Error>().is_some() => {
```

Comment:
I know, the ability for the services to return an error to the client that originates in the localdb websocket server is currently pending

I'm aware that removing and not replacing this code is not good, but I was at a loss when I last tried to implement it, and this would have been dead code if I had left it

That said, I could still try to implement it for this change!

Reply:
It's fine here, as the database doesn't have a procedural function implementation